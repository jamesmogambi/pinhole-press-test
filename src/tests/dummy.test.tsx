import React from "react";
import { shallow } from "enzyme";

const Dummy: React.FC<{ title: string }> = ({ title }) => <h1>{title}</h1>;

test("renders dummy component correctly", () => {
  let title = "Dummy component";
  let wrapped = shallow(<Dummy title={title} />);
  expect(wrapped).toMatchSnapshot();
});
