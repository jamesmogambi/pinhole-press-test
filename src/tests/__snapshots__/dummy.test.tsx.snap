// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders dummy component correctly 1`] = `
<h1>
  Dummy component
</h1>
`;
